Source/Core/Editor/UI/Windows/StepWindow.cs

```diff
@@ -31,18 +38,23 @@ internal class StepWindow : EditorWindow, IStepView
         /// </summary>
         public static void ShowInspector()
         {
-            StepWindow window = GetInstance();
+            StepWindow window = GetInstance(); //we are not using GetInstance(true) because of an issue with duplicated step inspectors PR#89
             window.Repaint();
-            window.Focus();
+            if (focusWindow)
+            {
+                window.Focus();
+            }
         }
-
         public static StepWindow GetInstance(bool focus = false)
         {
+            focusWindow = true;
             return GetWindow<StepWindow>("Step Inspector", focus);
+
         }
 
         private void OnEnable()
         {
+            focusWindow = false;
             GlobalEditorHandler.StepWindowOpened(this);
```

Comment:
Won't this end up calling ShowInspector, which in turn calls GetInstance and sets focusWindow to true again, making the bool useless? What am I missing?

Reply:
You are right! I'm sure it was working and I don't know wahy I tested it several times. Now it does not ._.
I feel like a junior dev

Reply:
@VaLiuM09 I implemented a new solution what do you think. Please test to be sure. 
I also tried to use [RuntimeInitializeOnLoadMethod(RuntimeInitializeLoadType.AfterAssembliesLoaded)] but di not work and confusingly was called after OnEnable. I did not further investigate as I cam up with the reset variable on domain reload idea.